app/controllers/assessment/grading.rb

```diff
@@ -286,8 +286,9 @@ def submission_popover
   end
 
   def score_grader_info
-    grader = Score.find(params[:id]).grader
-    if grader
```

Comment:
@icanberk ruby if statements don't need to have a then unless the block is on the same line (if true then blah end), but I think it's good practice to have then's at the end of if statements.